### DISCUSSION TITLE
I want to contribute - any good first issues / features to tackle?

### DISCUSSION THREAD

**Discussion by ggomaeng** [wevm/frog-archived#188] [Q&A] [answered]
I genuinely ♥️ frog 🐸 & everything else wevm does in the ecosystem.

I would love to contribute more to the growth of frog framework and frames ecosystem; however this is my **first time contributing to an open source project**, and I want to know if there would be any good first features or issues that I could tackle.

Sometimes I feel lost and worried that I might be making something that would be unnecessary & hurts the framework design. It would save my time as well as the core devs' time who are reviewing my PRs.

Any guidance would be greatly appreciated 🙏

**Comment by tmm** [comment, marked as answer]
Any of the issues are a good place to start. You are also welcome to improve test coverage or documentation. Finally, if you think of anything that would be a nice addition, feel free to create a new discussion briefly outlining what you want to do and we can take a look.
